go.mod

```diff
@@ -43,11 +43,13 @@ require (
 	k8s.io/klog/v2 v2.9.0
 	k8s.io/kubectl v0.22.2
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
-	sigs.k8s.io/cluster-api v1.0.0
-	sigs.k8s.io/cluster-api/test v1.0.0
-	sigs.k8s.io/controller-runtime v0.10.2
+	sigs.k8s.io/cluster-api v0.0.0-20211028170527-10d89ceca938
```

Comment:
/hold

until this is in a CAPI release

Reply:
backport pr https://github.com/kubernetes-sigs/cluster-api/pull/5541